packages/react-ui-ag/src/Drawer/__tests__/Drawer-tests.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import ThemedDrawer from '../Drawer'
+import theme from './mocks/theme'
+
+const Drawer = ThemedDrawer.WrappedComponent
+
+describe('Drawer', () => {
+  it('renders the base class and children', () => {
+    const props = {
+      theme,
+      className: theme.Drawer,
+      children: (<div>Content</div>),
+    }
+
+    const wrapper = mount(<Drawer {...props} />)
```

Comment:
Any reason this can't be shallow? I see a lot places in here where it seems doable with shallow.

Reply:
I am testing for the toggle-button. These tests seem to fail when I use shallow